operations/deployment/terraform/security-group.tf

```diff
@@ -8,7 +8,7 @@ resource "aws_security_group" "ec2_security_group" {
     cidr_blocks = ["0.0.0.0/0"]
   }
   tags = {
-    Name = "${var.aws_resource_identifier}"
+    Name = "${var.aws_resource_identifier}-sg"
```

Comment:
probably shouldn't add `-sg` here as it might fall into the same `-tf-state` issue we're having.  I think adding `-sg` to a security group might be redundant anyway whereas with `-tf-state`, the bucket could be anything, so it makes sense to add more context to the name